src/kf.jl

```diff
@@ -19,6 +19,18 @@ function update(filter::AbstractFilter, b0::GaussianBelief,
     return bn
 end
 
+function update_with_info(filter::AbstractFilter, b0::GaussianBelief,
```

Comment:
Should just be `update_info`

Reply:
Doesn't `update_info` suggest that the function is simply updating some information? However, the function is actually updating the filter and returning the updated `GaussianBelief` with information

Reply:
Yes, that is a valid objection, and I am glad that it bothers you. I suggested `update_info` because it matches `POMDPs.update_info`. I do not remember exactly why we chose that over `update_with_info`, probably simply for conciseness. In any case, this is a subjective choice, so you can decide what to do.

Reply:
Definitely `measure_info` and `update_with_info` should have the same form. A docstring can easily explain what the function does.